blueprints/app/files/_ts_eslint.config.mjs

```diff
@@ -38,7 +38,6 @@ export default tseslint.config(
   eslint.configs.recommended,
   eslintPluginEmber.configs.base,
   eslintPluginEmber.configs.gjs,
-  eslintPluginEmber.configs.gts,
```

Comment:
Redundant (declared below), if not, possibly incorrect as it may affect `*.{js,gjs}` if defined at a global level.